### DISCUSSION TITLE
Use stock iOS share sheet by default

### DISCUSSION THREAD

**Discussion by cammdavie** [qnblackcat/uYouPlus#707] [Ideas]
I was wondering if I could request this feature. I didn't see any discussions talking about the share sheet and also couldn't find any settings in the app for changing the share button behavior.  

The custom sharesheet that youtube provides is annoying to use because it doesn't have the recent contacts list that's available on the stock iOS share sheet (which is still available but requires hitting the more button at the end of the youtube share list). It would be nice to be able to skip that middle step.

**Comment by yodaluca23** [comment]
I agree it's a super annoying middle step.

**Comment by yodaluca23** [comment]
Ok, theirs a tweak made by iCraze, that does this, but I doubt it works anymore, but someone could experiment around with it maybe?
https://www.ios-repo-updates.com/repository/icraze-s-repo/package/com.icraze.ytstockshare/
